create-headless-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Since all this really does is create a docker image, why not name it something like that. Ideally, we should emulate how we build (or even publish) docker images for other repos. See for example the scripts/docker-build.* files under killrvideo-generator: https://github.com/KillrVideo/killrvideo-generator/tree/master/scripts. 

In fact, starting to think we should just go ahead and generate docker images for all of the service implementations, because why not? We should probably do this work in other PRs.

Reply:
Hrmmm...Yea, good point on the naming.  To your second point about generating docker images for all the service implementations unless I am misunderstanding your question we already do this.  Each are generated and then stored up on Docker Hub.  These are then pulled down when running docker-compose, no need to generate them locally.  Killrvideo-java is the only one we don't do this with mainly because the "headless" version was really something only we use for our cloud deployment.  I never really intended that to be publicly available as the intent is for folks to run Killrvideo-java via their IDE from a dev perspective.